### PR TITLE
Add Reviews for the restaurant detail page

### PR DESCRIPTION
**Pull request by syed-tp**
### Integration of Review Model into the Restaurant Detail Page

- **Created a Review Form:** Developed a form to allow users to submit reviews for restaurants, including fields for rating and comments.

- **Defined URL Path for Adding Reviews:** Added a new URL pattern to handle the submission of reviews, facilitating easy navigation to the review functionality.

### CODE REVIEW

File: restaurants/views.py [hunk 1]
```diff
@@ -23,16 +28,35 @@ def get_queryset(self):
...
-   
+
+class ReviewCreateView(LoginRequiredMixin, CreateView):
+    model = Review
+    form_class = ReviewForm
+    template_name = 'restaurants/review_form.html'
+
+    def form_valid(self, form):
+        form.instance.user = self.request.user
+        form.instance.restaurant_id = self.kwargs['id']
```
Comment (shantanutomarr) [inline restaurants/views.py]: This will raise unhandled error if some invalid restaurant id is provided in the URL. Before passing the data to the form, validate that the restaurant exists. Raise 404 if it does not.

You can do the following in dispatch() method after overriding.
`self.restaurant = get_object_or_404(Restaurant, pk=self.kwargs['id'])`

File: static/css/base.css [hunk 2]
```diff
@@ -238,7 +238,7 @@ input:checked + .slider:before {
     align-items: center;
     margin: 10px 0;
     font-family: 'Helvetica Neue', Arial, sans-serif;
-    font-size: 12px; /* Smaller font size */
```
Comment (shantanutomarr) [inline static/css/base.css]: Please avoid these cosmetic changes in the PR. These changes are unrelated to this PR.

You can make these kinds of changes directly to master without review. For example, removing comments, removing/adding empty lines, removing unused imports, etc.

Comment (syed-tp) [reply static/css/base.css]: Noted sir

**Review by shantanutomarr — APPROVED** [review]
